modin/pandas/test/test_api.py

```diff
@@ -67,7 +67,6 @@ def test_top_level_api_equality():
         "os",
         "multiprocessing",
         "Client",
-        "dask_client",
```

Comment:
@anmyachev, could you check this list for other options to be removed? I think we can delete something else.